docs/data/data-grid/export/export.md

```diff
@@ -220,16 +224,65 @@ The demo below shows how to add a JSON export.
 
 {{"demo": "CustomExport.js", "bg": "inline", "defaultCodeOpen": false}}
 
-> ⚠️ Due to the fact that the Print export relies on the usage of an `iframe`, there is a limitation around the usage of `X-Frame-Options`.
->
-> In order for the Print export to work as expected set `X-Frame-Options: SAMEORIGIN`.
+## Excel export [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
 
-## 🚧 Excel export [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
+The Excel export allows translating columns' type and tree structure of a DataGrid to an Excel file.
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #198](https://github.com/mui/mui-x/issues/198) if you want to see it land faster.
-> You will be able to export the displayed data to Excel with an API call, or using the grid UI.
+Columns with types `'boolean'`, `'number'`, `'singleSelect'`, `'date'`, and `'dateTime'` are exported in their corresponding type in Excel. Please ensure the `rows` values have the correct type, you can always [convert them](/components/data-grid/columns/#converting-types) as needed.
+
+{{"demo": "ExcelExport.js", "bg": "inline", "defaultCodeOpen": false}}
+
+This feature relies on [exceljs](https://github.com/exceljs/exceljs).
+To install it:
+
+```sh
+ // with npm
+ npm install exceljs
+
+ // with yarn
+ yarn add exceljs
+```
+
+### Customizing the exported file
```

Comment:
Maybe to keep it one line?

```suggestion
### Customizing the export
```
<img width="224" alt="Screenshot 2022-04-04 at 23 58 38" src="https://user-images.githubusercontent.com/3165635/161639127-8526d6c8-5768-4be9-aea6-c6049eb689d6.png">